### PR TITLE
Added different values to test the std

### PR DESCRIPTION
**Pull request by RCvanOosterhoudt**
#1

### CODE REVIEW

**Comment by xinyiwan** [comment]
👍

File: tests/test_models.py [hunk 1]
```diff
@@ -96,3 +96,14 @@ def test_daily_std():
     test_result = np.array([0, 0])
 
     npt.assert_array_equal(daily_std(test_input), test_result)
+
+
+def test_daily_std_values():
```
Comment (abdullahthabit) [inline tests/test_models.py:101]: You forgot to add the docstring

File: tests/test_models.py [hunk 2]
```diff
@@ -96,3 +96,14 @@ def test_daily_std():
...
+    from inflammation.models import daily_std
+
```
Comment (abdullahthabit) [inline tests/test_models.py]:
```suggestion
"""This is a test function with positive integers for the std-dev function """
```

**Review by abdullahthabit — CHANGES_REQUESTED** [review]

**Review by abdullahthabit — APPROVED** [review]

**Review by xinyiwan — APPROVED** [review]
LGTM

**Pull request merged**